R/Spectra.R

```diff
@@ -1466,6 +1465,7 @@ setMethod("pickPeaks", "Spectra",
                             halfWindowSize = halfWindowSize, method = method,
                             snr = snr, k = k, descending = descending,
                             threshold = threshold, msLevel = msLevel.)
+    object$centroided[msLevel(object) %in% msLevel.] <- TRUE
```

Comment:
I am still not convinced to set this "undelayed" (immediately). If we chain functions in the processing queue that depend on `centroided == FALSE` before `pickPeaks` they will fail (currently we don't have one but who knows).